app/models/rules_of_origin/heading_mappings.rb

```diff
@@ -7,7 +7,7 @@ class HeadingMappings
     SERVICES = %w[uk xi both].freeze
     SUB_HEADING_FORMAT = %r{\A\d{6}\z}
     DEFAULT_SOURCE_PATH = Rails.root.join('db/rules_of_origin').freeze
-    DEFAULT_FILE = 'rules_to_commodities_210929.csv'
+    DEFAULT_FILE = 'rules_to_commodities_211124.csv'
```

Comment:
Given we change these files so frequently is there any appetite for decoupling the uploading of the files from the codebase?

We still get the persistence buckets for dev, staging and production so we can verify them first

Reply:
That's a good point, this is done, but happy to discuss it next time we must update those rules.

Reply:
Sure :)